docs/opentelemetry.rst

```diff
@@ -0,0 +1,102 @@
+OpenTelemetry: Monitoring Performance
+=====================================
+
+What is OpenTelemetry?
+----------------------
+
+`OpenTelemetry <https://opentelemetry.io>`_ is an open-source observability framework for `distributed traces <https://uptrace.dev/opentelemetry/distributed-tracing.html>`_, `metrics <https://uptrace.dev/opentelemetry/metrics.html>`_, and logs.
+
+OpenTelemetry allows developers to collect and export telemetry data in a vendor agnostic way. With OpenTelemetry, you can instrument your application once and then add or change vendors without changing the instrumentation, for example, here is a list of `popular DataDog alternatives <https://uptrace.dev/get/compare/datadog-competitors.html>`_ that support OpenTelemetry.
+
+OpenTelemetry instrumentation
+-----------------------------
+
+Instrumentations are plugins for popular frameworks and libraries that use OpenTelemetry API to record important operations, for example, HTTP requests, DB queries, logs, errors, and more.
+
+To install OpenTelemetry `instrumentation <https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/redis/redis.html>`_ for redis-py:
+
+.. code-block:: shell
+
+   pip install opentelemetry-instrumentation-redis==0.35b0
+
+You can then use it to instrument code like this:
+
+.. code-block:: python
+
+   from opentelemetry.instrumentation.redis import RedisInstrumentor
+
+   RedisInstrumentor().instrument()
+
+Once the code is patched, you can use redis-py as usually:
+
+.. code-block:: python
+
+   # Sync client
+   client = redis.StrictRedis(host="localhost", port=6379)
+   client.get("my-key")
+
+   # Async client
+   client = redis.asyncio.Redis(host="localhost", port=6379)
```

Comment:
Actually, also defaults are localhost/6379

Reply:
Yeah, but my thinking is that with explicit values users can more easily copy-paste the example and change credentials. WDYT?